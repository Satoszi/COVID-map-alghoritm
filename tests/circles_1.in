6 6 2 11
circles_1
+----+
|+--+|
||++||
||++||
|+--+|
+----+