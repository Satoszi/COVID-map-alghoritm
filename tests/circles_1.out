1 5
5 4
4 4
1 4
0 3
2 2
5 1
4 1
1 1
3 0
0 0
